src/Catalog/NuGet.Services.Metadata.Catalog.csproj

```diff
@@ -197,9 +197,6 @@
     <EmbeddedResource Include="sparql\ConstructPackagePageContentGraph.rq" />
   </ItemGroup>
   <ItemGroup>
-    <Content Include="content\further.js" />
```

Comment:
How did we verify that these are not used?

Reply:
I can't find any references to them.

Looking at the git history, it looks like they were used in a [2014 catalog implementation](https://github.com/NuGet/NuGet.Services.Metadata/commit/67433f53244456e1f4656c68cf1325bcbc1327e1), but have likely not been used for years.